test/integration/scheduler/extender_test.go

```diff
@@ -33,6 +33,7 @@ import (
 	"k8s.io/kubernetes/pkg/api/resource"
 	"k8s.io/kubernetes/pkg/api/testapi"
 	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
```

Comment:
> everywhere else this import is renamed to clientset

Internal versus external is going to matter as we start trying to use the external clients we're vendoring in.  I'd like to start making the distinction.